src/components/Dashboard/LatestActivity.js

```diff
@@ -12,30 +12,31 @@ function LatestActivity() {
   return (
     <Box heading="latest activity" padding={0}>
       <Stepper
-        lineColor="rgba(255, 151, 128, 0.2)"
+        lineColor={theme.accent.alpha(0.2)}
         lineExtraHeight={5}
         lineTop={15}
         lineWidth={2}
         css={`
           padding: ${3 * GU}px 0;
         `}
       >
-        {latestActivity.map(activity => (
+        {latestActivity.map((activity, index) => (
           <Step
+            key={index}
             stepPoint={
               <div
                 css={`
                   border-radius: 50%;
-                  border: 2px solid rgb(255, 151, 128, 0.2);
-                  padding: 8px;
+                  border: 2px solid ${theme.accent.alpha(0.2)};
+                  padding: ${1 * GU}px;
                 `}
               >
                 <span
                   css={`
-                    background: rgb(255, 151, 128);
-                    border-radius: 50%;
                     display: block;
-                    padding: 5px;
+                    padding: ${0.5 * GU}px;
```

Comment:
Just a question, how do you guys do when for example on a figma file says 5px of padding ? we just say ok let's use the nearest value using the grid unit? in this case 0.5 = 4px?

Reply:
It depends haha :grimacing: 

We generally try to understand if it could work using the nearest 2 / 4 / 8 or if the value is really intended. For normal spacing like in this case, it’s generally safe to assume that it’s a mistake in the Figma file and we can correct it ourselves.